Mathlib/Data/Multiset/Basic.lean

```diff
@@ -2327,6 +2327,24 @@ theorem map_count_True_eq_filter_card (s : Multiset α) (p : α → Prop) [Decid
   simp only [count_eq_card_filter_eq, filter_map, card_map, Function.id_comp,
     eq_true_eq_id, Function.comp_apply]
 
+@[simp] theorem sub_singleton [DecidableEq α] (a : α) (s : Multiset α) : s - {a} = s.erase a := by
+  ext
+  simp [Multiset.erase_singleton, Multiset.count_singleton]
```

Comment:
`simp only ...`?

Reply:
I think `split` should be considered a fancy tactic here

Reply:
What does "fancy" mean here? That we do not need to change this to `simp only`?

Reply:
Yes, basically. See eg https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Terminal.20tidy/near/293210921

Reply:
I disagree here, I've seen plenty on nonterminal simps before a split break.

Reply:
I pushed a change for this.